client/src/routes/screens/Home/Home.tsx

```diff
@@ -217,6 +224,23 @@ const Home = () => {
     navigate('AboutOverlay');
   }, [navigate]);
 
+  const throttledRefresh = useMemo(
+    () => throttle(() => refreshPull(), 5 * 60000),
```

Comment:
How about only doing a `fetchSessions` and not `refreshPull` here since it will pull down and show that it's loading otherwise? Less intrusive imo

Reply:
Also shouldn't this be a useCallback? E.g.
```javascript
const throttledFetch = useCallback(throttle(fetchSessions), 5 * 60000), [fetchSessions])
```

Reply:
Nice! I like the less intrusive perspective, done [6479f65](https://github.com/29ki/29k/pull/2804/commits/6479f6536378430f562f65a514d08eac4ecd623c)